### ISSUE TITLE
Icons not showing in collapsed Menu

### ISSUE THREAD

**Issue by Akulatraxas** [sni/Thruk#1246]
**Describe the bug**
Icons in the collapsed view are not shown.

Its probably something with my installation but i cant figure out what it might be. 
When i observe the developer console i don't see any missing requests or errors. 
I tried Chrome, Firefox, Safari all with the same result.

**Thruk Version**
Thruk 3.02

**To Reproduce**
1. Open Thruk
2. Set your Menue to "Collapsed"
3. Icons in the Menu are all missing

**Expected behavior**
Icons are shown

**Screenshots**
![image](https://user-images.githubusercontent.com/103323/215073303-ca76a87b-b621-47f9-8af8-effee2dc462f.png)
![image](https://user-images.githubusercontent.com/103323/215073575-d681093e-6e95-4cee-b638-0fa2ed0d9677.png)

**Comment by sni** [comment]
The collapsed menu uses the same icons as the other pages. All icons are font based and either from the unicon or the font-awesome set.
You can have a look at the "themes.cgi" to see a list of all icons. The themes.cgi is not in the menu, you have to enter the /thruk/cgi-bin/themes.cgi url manually.

**Comment by Akulatraxas** [comment]
In the themes.cgi the icons are loading properly. They are just missing in the collapsed menu:
Any idea what i can do to narrow down the problem ? 

![image](https://user-images.githubusercontent.com/103323/215092874-40c8a463-c67b-47e8-b21e-f7aa35b43a7b.png)

**Comment by Akulatraxas** [comment]
I noticed that when comparing our ui with the one on the demo page that the uil-* icon definitions are missing. 

Ours:
![image](https://user-images.githubusercontent.com/103323/215120857-849811f9-6de9-4a88-aa17-ee0a8ed193b8.png)

Demo:
![image](https://user-images.githubusercontent.com/103323/215121072-9c2da217-6beb-4dcc-9b05-122b8119f370.png)

After grepping a bit around i found the name uil-estate in the menu.conf. 
Our thruk was updated version a 2.x version and apparently it left the menu_local.conf untouched (i cant remember editing it). There was no icon definition in there which lead to the missing icons. Its fixed now :)
